### ISSUE TITLE
“Inbound” option available though Asimba is not installed

### ISSUE THREAD

**Issue by wasiqulhuq** [GluuFederation/oxTrust#889]
1. Login to Gluu
2. Click on SAML
3. Observe Outbound and Inbound links 

Check below screen,
![saml](https://user-images.githubusercontent.com/36174217/37120813-9dabacfa-2285-11e8-9316-ca0e4d66a77e.jpg)

**Comment by wasiqulhuq** [comment]
```
GLUU.root@wasiqul:~# cat setup.properties.last | grep -i installAsimba
cat: setup.properties.last: No such file or directory
```

**Comment by shekhar16** [comment]
Duplicate issue 
Fixed in #853
